### PR TITLE
docs: Document the additional parameters in JWT

### PR DESCRIPTION
**Pull request by ifly7charlie**
Expose the signingKey, and options for verification and encryption

**What**: Documentation of signingKey and options

**Why**: To make it easier to control the JWT key

**How**: Extra documentation text

**Checklist**:

- [ x ] Documentation
- [ ] Tests
- [ x ] Ready to be merged

### CODE REVIEW

File: www/docs/configuration/options.md [hunk 1]
```diff
@@ -121,11 +121,29 @@ By default JSON Web Tokens are signed (JWS) but not encrypted (JWE), as JWT encr
 jwt: {
   // A secret to use for key generation - you should set this explicitly
   // Defaults to NextAuth.js secret if not explicitly specified.
+  // This is used to generate the actual signingKey and produces a warning
+  // message.
   // secret: 'INp8IvdIyeMcoGAgFGoA61DdBglwwSqnXJZkgz8PSnw',
   
+  // You can specific a signing key generated using jose newkey -s 512 -t oct -a HS512
+  // This gives you direct knowledge of the key used to sign the token so you can use it
+  // to authenticate indirectly (eg to a database driver)
```
Comment (balazsorban44) [inline www/docs/configuration/options.md:130]:
```suggestion
  // You can generate a signing key using `jose newkey -s 512 -t oct -a HS512`
  // This gives you direct knowledge of the key used to sign the token so you can use it
  // to authenticate indirectly (eg. to a database driver)
```

File: www/docs/configuration/options.md [hunk 2]
```diff
@@ -121,11 +121,29 @@ By default JSON Web Tokens are signed (JWS) but not encrypted (JWE), as JWT encr
 jwt: {
   // A secret to use for key generation - you should set this explicitly
   // Defaults to NextAuth.js secret if not explicitly specified.
+  // This is used to generate the actual signingKey and produces a warning
+  // message.
```
Comment (balazsorban44) [inline www/docs/configuration/options.md:125]:
```suggestion
  // This is used to generate the actual signingKey and produces a warning
  // message if not defined explicitly.
```

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
Thank you! Could you please address my small changes, and this will be good to go!

**Comment by ifly7charlie** [comment]
Thank you for that - it's embarrassing how wrong the text was!

**Review by balazsorban44 — APPROVED** [review]
@ifly7charlie thank you so much! Honest contributions are never embarrassing! Happy for your help here!

**Pull request merged**